ansible/noto.yml

```diff
@@ -1,4 +1,16 @@
 ---
+- name: Patch yum sources
+  hosts: manager:acs
```

Comment:
This patch should be applied to all acs machine, not sure about the : syntax, if it works it's ok to be kept as it is

Reply:
The ":" syntax allows to target multiple machines